packages/minizip/ioapi.c

```diff
@@ -220,8 +220,8 @@ static int ZCALLBACK ferror_file_func (voidpf opaque, voidpf stream)
     return ret;
 }
 
-void fill_fopen_filefunc (pzlib_filefunc_def)
-  zlib_filefunc_def* pzlib_filefunc_def;
```

Comment:
This file is third party code, so we should not change it but request the change from minizip maintainers. Nevertheless, this going to take too much time, and I'm not sure if this is still maintained.. So I would suggest to comment out the removed line and include a comment for future reference, so that when comparing a future version of ioapi.c (probably never) with current one we will spot the change.

Please review and let me know what you think